docs/3-create-a-zarf-package/6-package-sboms.md

```diff
@@ -0,0 +1,37 @@
+# Package SBOMs
+
+<!-- TODO (@WSTARR) REWRITE -->
```

Comment:
Did you finish the rewrite you were wanting to do?

Reply:
maybe... lol